magento/magento2
# 2.3 only - creating attribute option value using API returns unexpected response #18392

### Description (*)
Return value should be the ID of the new option

### Fixed Issues (if relevant)

1. magento/magento2#18392: 2.3 only - creating attribute option value using API returns unexpected response

### Manual testing scenarios (*)

1. Setup a clean Magento installation
2. Create an Integration with full access rights & activate it, I'm assuming the access token is abcd (see step 4)
3. Using the adminhtml, create a new dropdown product attribute, use attribute code: dropdown
4. Add an option through a REST call (make sure to use an option which doesn't exist yet):
`curl -k -X POST "https://domain.tld/rest/all/V1/products/attributes/dropdown/options" -H "accept: application/json" -H "Content-Type: application/json" -H "Authorization: Bearer abcd" -d "{ \"option\": { \"label\": \"whatever\" }}`
5. Value should be the ID of the new option

### Contribution checklist (*)
